src/orion/plotting/backend_plotly.py

```diff
@@ -604,37 +697,57 @@ def get_objective_name(experiments):
                 return trials[0].objective.name
         return "objective"
 
-    ORDER_KEYS = ["suggested", "reserved", "completed"]
+    ORDER_KEYS = ["suggested", "reserved", "completed", "objective"]
 
     if not experiments:
         raise ValueError("Parameter 'experiment' is None")
 
     if order_by not in ORDER_KEYS:
         raise ValueError(f"Parameter 'order_by' is not one of {ORDER_KEYS}")
 
-    df = build_groups()
-
+    df, x_col, y_col, x_title = build_groups()
     fig = go.Figure()
-
     if df.empty:
         return fig
 
     names = set(df["name"])
     for i, name in enumerate(sorted(names)):
         exp_data = df[df["name"] == name]
-        if "best_mean" in exp_data:
-            y = exp_data["best_mean"]
+
+        y = exp_data[y_col]
+        x = exp_data[x_col]
+
+        duration_col = "duration_mean" if "duration_mean" in exp_data else "duration"
+        durations = exp_data[duration_col].tolist()
+        unit_time = durations[-1] / len(durations)
+        if unit_time > 60 * 60 * 24:
```

Comment:
Factoring out this part as a function would make it easy to test it separately. I think we should have a min number of units to select seconds, minutes, etc. If we select minute instead of seconds for instance, there should be at least 3 of 4 minutes so that the tick labels are > 1. Otherwise it would hurt readability. The factors (ex: 60 * 60 * 24) could be multiplied by this min number of units.

Reply:
You mean factoring the code to compute `duration `and `time_unit` as a function? 
Let's make the min number of unit as 3.

Reply:
Yep, having a function like this `unit_time = infer_unit_time(durations, min_unit=3)`

Reply:
updated in https://github.com/Epistimio/orion/pull/685/commits/884eaffee26bd0d91ff7fd81e67cc24efb9e7132

Reply:
Looks like the test did not cover this case: https://github.com/Epistimio/orion/pull/685/commits/884eaffee26bd0d91ff7fd81e67cc24efb9e7132#diff-3de0162ba5d54eda9f327045182609e93571e11db5ad711067d7c54c02155d52R633. Otherwise it's good! :)